style/swift/README.md

```diff
@@ -5,6 +5,7 @@ Swift
 
 * Keep up with the Objective-C style guide above. Will highlight differences
   here.
+* Limit lines to 100 characters
```

Comment:
We work with a lot of older API from ObjC that have very verbose names. Sometimes this is unrealistic.